Subject: [PATCH] Remove backtrace from test.c

This is because backtrace doesn't exist
in Android NDK by the looks of it since
it's failing with compile warnings.
Because it's in test file, change can be applied
globally for all builds.
---
 test/test.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/test/test.c b/test/test.c
index 039971b..de88765 100644
--- a/test/test.c
+++ b/test/test.c
@@ -37014,12 +37014,8 @@ void test_StanSubTimeout(void)
 
 #ifndef _WIN32
 static void _sigsegv_handler(int sig) {
-  void *array[20];
-  int size = backtrace(array, 20);
-
   // print out all the frames to stderr
   fprintf(stderr, "Error: signal %d:\n", sig);
-  backtrace_symbols_fd(array, size, STDERR_FILENO);
   exit(1);
 }
 #endif // _WIN32
-- 
2.45.2.windows.1

